internal/common/requests.go

```diff
@@ -1,10 +1,11 @@
 package common
 
 type Publisher struct {
-	CodeHosting []CodeHosting `json:"codeHosting" validate:"required"`
-	Description string        `json:"description"`
-	Email       string        `json:"email" validate:"email"`
-	Active      bool          `json:"active"`
+	CodeHosting  []CodeHosting `json:"codeHosting" validate:"required,gt=0,dive"`
+	Description  string        `json:"description"`
+	Email        string        `json:"email" validate:"email,required"`
```

Comment:
`required` won't work for `application/merge-patch+json`, because it can be omitted

Take a look at https://github.com/italia/developers-italia-api/pull/138, which has two different structs for POST and PATCH

Reply:
You're right.